workflow/common/jsonpatch.go

```diff
@@ -0,0 +1,7 @@
+package common
+
+type Operation struct {
```

Comment:
This could have a better name. `common.Operation` doesn't describe it very well. Perhaps 'DeleteFinalizerPatch`? There could be a helper function in here to json.Marshal it too.